packages/react/src/lib/renderComponent.tsx

```diff
@@ -155,7 +155,10 @@ const renderComponent = <P extends {}>(config: RenderConfig<P>): React.ReactElem
     <FelaTheme>
       {(theme: ThemePrepared) => {
         if (_.isEmpty(theme)) {
-          logProviderMissingWarning()
+          if (className !== 'ui-provider') {
```

Comment:
```suggestion
          if (_.isEmpty(theme) && className !== 'ui-provider') {
```

Reply:
As `Provider` is no longer a UIComponent I reverted this change.